src/test/javascript/spec/service/exercise-update-warning.service.spec.ts

```diff
@@ -42,10 +38,17 @@ describe('Exercise Update Warning Service', () => {
         expect(updateWarningService.instructionDeleted).to.equal(true);
     });
 
-    it('should set scoringChanged as true', () => {
+    it('should set creditChanged as true', () => {
         exercise.gradingCriteria = [gradingCriterionCreditsChanged];
         backupExercise.gradingCriteria = [gradingCriterion];
         updateWarningService.loadExercise(exercise, backupExercise);
-        expect(updateWarningService.scoringChanged).to.equal(true);
+        expect(updateWarningService.creditChanged).to.equal(true);
+    });
+
+    it('should set usageCountChanged as true', () => {
+        exercise.gradingCriteria = [gradingCriterionUsageCountChanged];
+        backupExercise.gradingCriteria = [gradingCriterion];
+        updateWarningService.loadExercise(exercise, backupExercise);
+        expect(updateWarningService.usageCountChanged).to.equal(true);
```

Comment:
I think this can be moved to a function since all three test use the same structure.